examples/model_hosting/schedule_inspection/CPR/setup.py

```diff
@@ -0,0 +1,5 @@
+from setuptools import find_packages, setup
+
+REQUIRED_PACKAGES = ["cognite-model-hosting==0.1.3"]
```

Comment:
Lock this to 0.1.* instead, so we don't have to update it if we release patches.